core/src/main/java/org/apache/iceberg/BaseTableScan.java

```diff
@@ -273,7 +269,7 @@ public String toString() {
     return MoreObjects.toStringHelper(this)
         .add("table", table)
         .add("projection", schema().asStruct())
-        .add("filter", context.rowFilter())
+        .add("filter", context.filterRows())
```

Comment:
The `rowFilter` getter doesn't need to change. It is still getting the row filter. Just the setter should be a verb.